packages/vanilla/src/components/card/decisionCard.stories.js

```diff
@@ -118,21 +119,47 @@ const ProfileCardTemplate = ({ title, danger, actionsLayout }) => {
           </p>
         </div>
       </div>
-      ${renderActions(actionsLayout)}
+      ${renderActions(actionsLayout, actionsConfig)}
     </div>
   `;
 };
 
+const actionsConfig = {
+  btn1: {
+    label: 'Label',
+    type: 'primary',
```

Comment:
This control is "color" in the buttons stories, so I would use "color" instead of "type", which is a reserved attribute on buttons anyway.